data/secrets.yml

```
@@ -0,0 +1,37 @@
+env:
```

- https://github.com/meltano/kms-ext/pull/2

```suggestion
kms_key_id: 17dcca7a-39d3-46b1-b2b3-83fcc2375747
env:
```

This will help quickly address this issue:
- https://github.com/meltano/infra/issues/140